client/src/fabric/FabricRuntimeManager.ts

```diff
@@ -33,9 +33,9 @@ export class FabricRuntimeManager {
 
     private static _instance: FabricRuntimeManager = new FabricRuntimeManager();
 
-    private connectionRegistry: FabricGatewayRegistry = FabricGatewayRegistry.instance();
     private runtimeRegistry: FabricRuntimeRegistry = FabricRuntimeRegistry.instance();
     private runtimes: Map<string, FabricRuntime> = new Map<string, FabricRuntime>();
```

Comment:
There is now only one so don't need a map of runtimes, and we don't need a runtime registry, we should update the settings to just store an object for a runtime rather than an array